samples/ruby/example.rb

```diff
@@ -0,0 +1,22 @@
+require 'httparty'
+require_relative 'lib/api_auth'
+
+API_KEY = "57502612d1bb2c00010000256c9568f1f7d540e085052b425d46d233"
```

Comment:
Could you possibly update this to have a placeholder like in the Java Example such as < YOUR API_KEY >